front/app/containers/MainHeader/NotificationMenu/components/NotificationWrapper/index.tsx

```diff
@@ -113,11 +113,7 @@ const NotificationWrapper = ({
       </IconContainer>
       <Body>
         <Message isRead={isRead}>{children}</Message>
-        {timing && (
-          <Timing>
-            <FormattedRelative value={timing} />
-          </Timing>
-        )}
+        {timing && <Timing>{timeAgo(timing, 'en')}</Timing>}
```

Comment:
This 'en' needs to be replaced by the locale, right?

Reply:
timing here is a string, while in other places where we use timeAgo we Date.parse this string to become a number. Can't we do that here as well, so we can add the parameter types of timeAgo?